db/testing.py

```diff
@@ -68,5 +69,5 @@ def load_low_level_data(self, mbid):
         the database.
         """
         with open(self.data_filename(mbid)) as json_file:
-            db.data.submit_low_level_data(mbid, json.loads(json_file.read()))
+            db.data.submit_low_level_data(mbid, json.loads(json_file.read()), 'mbid')
```

Comment:
Same issue with type here. You should reference a variable. Something like `GID_TYPE_MBID`.